examples/src/main/java/com/expediagroup/sdk/xap/examples/services/LodgingService.java

```diff
@@ -47,7 +49,7 @@ public AvailabilityCalendarResponse getAvailabilityCalendars() {
         GetLodgingAvailabilityCalendarsOperationParams.builder()
             .partnerTransactionId(UUID.randomUUID().toString())
             //Comma-separated list of Expedia Property IDs.
-            .propertyIds("87704892,36238803")
+            .propertyIds(new HashSet<>(Arrays.asList("87704892", "36238803")))
```

Comment:
why do we need to wrap the array with a hashset?

Reply:
Because it's a `Set`, not a `List`, and `Set.of` is only available in Java 9+